js_modules/dagster-ui/packages/ui-core/src/metadata/MetadataEntry.tsx

```diff
@@ -30,6 +30,15 @@ import {NotebookButton} from '../ui/NotebookButton';
 import {DUNDER_REPO_NAME, buildRepoAddress} from '../workspace/buildRepoAddress';
 import {workspacePathFromAddress} from '../workspace/workspacePath';
 
+export const HIDDEN_METADATA_ENTRY_LABELS = new Set([
```

Comment:
Definitely see the case for hiding the last two in this list, but why hide the others?

Reply:
`dagster_dbt/select` and `dagster_dbt/exclude` are implementation-specific details (we're using the metadata not for display purposes, but to make the subsetting implementation more ergonomic).

It also doesn't really make sense when it's viewed in the asset sidebar. It's displayed on each individual asset generated by `@dbt_assets`, but its meaning only makes sense in the aggregate of the assets, at the op level where a single computation invoked `--select` or `--exclude`.

Reply:
I guess it might be more of an indicator to remove the "View metadata" button from the sidebar, since we do display that the metadata comes from the computation (e.g. `cloud_analytics_dbt_assets`, but it wasn't that obvious until I relooked at the screenshot

Reply:
I'm sold on hiding it. And it's a two-way door.